index.html

```diff
@@ -1997,6 +1997,47 @@ <h5>Get checksum for video/audio stream</h5>
     </div>
     <!-- ends Get checksum for video/audio stream -->
 
+    <!-- Get checksum for all video/audio streams -->
+    <label class="recipe" for="get_streamhash">Get individual checksums for all video/audio streams ("Streamhash")</label>
+    <input type="checkbox" id="get_streamhash">
+    <div class="hiding">
+      <h5>Get individual checksums for all video/audio streams ("Streamhash")</h5>
+      <p><code>ffmpeg -i <em>input_file</em> -map 0 -f streamhash -hash md5 - -v quiet</code></p>
+      <p>
+      The outcome is very similar to that of "-f hash", except you get one hash
```

Comment:
Nitpick - I would prefer if the text inside the `<p>` tags was indented inwards (also below)

Reply:
Sure!
You mean `<p>...</p>` instead of:
```
<p>
...
</p>
```

Will do that!

Reply:
In fact, I meant,
```html
<p>
  ....
  ....
</p>
```

Reply:
In my opinion, should be `<p>. . .</p>` without newlines inside the `. . .`.

Reply:
I feel like @retokromer gets final word on our style guide ;-), so let's go with his suggestion :-)